node/network/availability-distribution/src/lib.rs

```diff
@@ -0,0 +1,2030 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
Can this not be split up into multiple files? Reading a 2k file is really not that funny.

Reply:
It can be and it will be :) , I think tests can be pushed to a separate file quite easily, but let's see how that goes.